mtools/mplotqueries/plottypes/scatter_type.py

```diff
@@ -75,7 +75,7 @@ def plot_group(self, group, idx, axis):
         if self.logscale:
             axis.semilogy()
 
-        group_label = group.replace('$', '\$')
+        group_label = group.replace('$', r'\$')
```

Comment:
Just checking: we don't want to escape the "$", we want it to be replaced with the literal string "\$" ?

Reply:
@kallimachos This is the correct fix for the label rendering issue with the log snippet included on #613; will add that snippet to git.

Reply:
Cool.